api/v1alpha1/trafficmanagerbackend_types.go

```diff
@@ -75,18 +82,33 @@ type TrafficManagerEndpointStatus struct {
 	// +required
 	Name string `json:"name"`
 
+	// ResourceID is the fully qualified Azure resource Id for the resource.
+	// Ex - /subscriptions/{subscriptionId}/resourceGroups/{resourceGroupName}/providers/Microsoft.Network/trafficManagerProfiles/{profileName}/azureEndpoints/{name}
```

Comment:
this comment seems to indicate that the resourceID is the ATMs?

Reply:
yes, we'll show the Azure resource ID based on the wallow feedback for both profile and endpoint.

we'll continue to use uuid in ATM resource names to solve name conflicts issue.